Make PowerShell 5.1 usage more prominent (#7140)
open-telemetry/opentelemetry.io: content/en/docs/zero-code/dotnet/_index.md

aliases: [net]
redirects: [{ from: /docs/languages/net/automatic/*, to: ':splat' }]
weight: 30

> cSpell:ignore: coreutils HKLM iisreset myapp pwsh

Reviewer: ```suggestion
cSpell:ignore: coreutils HKLM iisreset myapp
```

### Windows (PowerShell)

{{% alert title="Note" color="warning" %}} PowerShell v6.0+ (pwsh) is not

> supported in install .NET zero-code instrumentation. The built-in version of

Author: ```suggestion
supported to install .NET zero-code instrumentation. The built-in version of
```

> {{% alert title="Note" color="warning" %}} PowerShell v6.0+ (pwsh) is not
> supported to install .NET zero-code instrumentation. The built-in version of
> Windows PowerShell must be used. {{% /alert %}}
> 
> On Windows, use the PowerShell module as an Administrator:

Reviewer: ```suggestion
On Windows, use the PowerShell module as an Administrator.

{{% alert title="Version note" color="warning" %}}

PowerShell v5.1 is required. In particular, v6.0+ is not yet supported.

{{% /alert %}}
```

Reviewer: I'll be OOO so I'll let @theletterf and @open-telemetry/dotnet-approvers follow up and merge when they see fit.

```powershell

> # PowerShell 5.1 is required

Reviewer: ```suggestion
#Requires -Version 5.1
```

### Windows (PowerShell)

On Windows, use the PowerShell module as an Administrator.

> {{% alert title="Version note" color="warning" %}} Windows
> [PowerShell Desktop](https://learn.microsoft.com/powershell/module/microsoft.powershell.core/about/about_windows_powershell_5.1#powershell-editions)
> (v5.1) is required. Other
> [versions](https://learn.microsoft.com/previous-versions/powershell/scripting/overview),
> including PowerShell Core (v6.0+) are not supported at this time.{{% /alert %}}

Reviewer 2: ```suggestion
{{% alert title="Version note" color="warning" %}} 

Windows
[PowerShell Desktop](https://learn.microsoft.com/powershell/module/microsoft.powershell.core/about/about_windows_powershell_5.1#powershell-editions)
(v5.1) is required. Other
[versions](https://learn.microsoft.com/previous-versions/powershell/scripting/overview),
including PowerShell Core (v6.0+) are not supported at this time.

{{% /alert %}}
```
